modules/create-policy-from-system-policies-view.adoc

```diff
@@ -8,58 +8,67 @@
 You can create new security policies from the system policies view.
 
 .Procedure
-. On the RHACS portal, navigate to *Platform Configuration* -> *Policies*.
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Policy Management*.
 . Click *Create policy*.
 . Enter the following details about your policy in the *Policy details* section:
 ** Enter a *Name* for the policy.
 ** Optional: Attach notifiers to the policy by selecting from the available *Notifiers* under the *Attach notifiers* section.
 +
 [NOTE]
 ====
-You must integrate {product-title} with your notification provider, for example, webhooks, Jira, PagerDuty, Splunk, or others before you can forward alerts.
+You must integrate {product-title-short} with your notification provider, for example, webhooks, Jira, PagerDuty, Splunk, or others before you can forward alerts.
 ====
 ** Select a *Severity* level for this policy, either `Critical`, `High`, `Medium`, or `Low`.
-** Select policy *Categories* you want to apply to this policy.
-** Enter details about the policy in the *Description* box.
-** Enter an explanation about why the policy exists in the *Rationale* box.
-** Enter steps to resolve violations of this policy in the *Guidance* box.
+** Select policy *Categories* you want to apply to this policy. For information about creating new categories, see "Creating and managing policy categories" later in this document.
+** Enter details about the policy in the *Description* field.
+** Enter an explanation about why the policy exists in the *Rationale* field.
+** Enter steps to resolve violations of this policy in the *Guidance* field.
 ** Optional: Under the *MITRE ATT&CK* section, select the link:https://attack.mitre.org/matrices/enterprise/containers/[tactics and the techniques] you want to specify for the policy.
-.. Click *Add tactic*, and then select a tactic from the dropdown list.
+.. Click *Add tactic*, and then select a tactic from the drop-down list.
 .. Click the *Add technique* to add techniques for the selected tactic. You can specify multiple techniques for a tactic.
 . Click *Next*.
-. In the *Policy behavior* section, select the *Lifecycle stages* and *Event sources (Runtime lifecycle only)* for the policy.
-** Choose *Lifecycle Stages* to which your policy is applicable, from *Build*, *Deploy*, or *Runtime*.
+. In the *Policy behavior* section, perform the following steps:
```

Comment:
```suggestion
. In the *Policy behavior* section, take the following steps:
```
IBM Style Word Usage guidance about _perform_:
>Do not use if more direct wording is possible. For example, instead of writing _Perform the product installation_, write _Install the product_.